libhybris/hybris/tests/test_gps.c

```diff
@@ -288,7 +288,7 @@ static void agps_handle_status_callback(AGpsStatus *status)
   {
     case GPS_REQUEST_AGPS_DATA_CONN:
         fprintf(stdout, "*** data_conn_open\n");
-#ifndef HAS_ANDROID_4_2_0
+#if ! defined(HAS_ANDROID_4_2_0) && ! defined(HAS_ANDROID_5_0_0)
```

Comment:
Should that not rather be || ? As i doubt 4_2_0 and 5_0_0 will be defined at the same time.